src/modules/shared/components/MediaCard/MediaCard.test.tsx

```diff
@@ -18,9 +15,7 @@ describe('Component: <MediaCard />', () => {
 	});
 
 	it('Should apply the vertical orientation when rendered in grid view', () => {
-		const element = documentOf(rendered).getElementsByClassName(
-			CardStyles['c-card--orientation-vertical']
-		);
+		const element = documentOf(rendered).getElementsByClassName('c-card--orientation-vertical');
```

Comment:
Ipv `documentOf(rendered)` te gebruiken kan je rechtstreeks de `container` aanspreken en een `querySelector`doen. 
`const { container } = render(<MediaCard />)`. 
Maar eigenlijk zou dit niet onze go to way mogen zijn om elementen op de halen. ([docs warning](https://testing-library.com/docs/react-testing-library/api/#container-1))

Reply:
Fixed in #41